ledger/internal/eval_test.go

```diff
@@ -759,6 +760,8 @@ func TestEvalFunctionForExpiredAccounts(t *testing.T) {
 		}
 		tmp := genesisInitState.Accounts[addr]
 		tmp.Status = basics.Online
+		crypto.RandBytes(tmp.StateProofID[:])
+		crypto.RandBytes(tmp.SelectionID[:])
```

Comment:
should we also randomize VoteID?

Reply:
I'm perfectly fine with randomizing the VoteID as well, but let's do that after the release ?
( i.e. since it's just a better testing )

Reply:
Gotcha !